hardhat/tasks/start-node.js

```diff
@@ -1,22 +1,39 @@
 require("@nomicfoundation/hardhat-toolbox");
 const { spawn } = require("child_process");
-const { forkPorts } = require("../constants/deploymentConfig");
+const { networkNameToPortName } = require("../helpers/functionHelpers");
 
 /**
  * @description Launch a node forking the specified network
- * @dev see constants/deploymentConfig::forkPorts for config of ports associated with networks
+ *
+ * @dev This task is used to fork a network and run a local node with the forked state.
+ * @dev Network & network'Fork' should be configured in hardhat.config.js
+ * @dev The port used for forking should be set in .env file
  */
 task("start-node", "Starts a Hardhat node with forking")
   .addParam("networkToFork", "The network to fork") // "forking"
   .setAction(async (taskArgs, hre) => {
+    const forkName = taskArgs.networkToFork + "Fork";
     const networkConfig = hre.config.networks[taskArgs.networkToFork];
     if (!networkConfig || !networkConfig.url) {
       throw new Error(
         `Network ${taskArgs.networkToFork} is not configured correctly.`
       );
     }
```

Comment:
`
if (!networkConfig?.url) {
  throw new Error(`Network '${taskArgs.networkToFork}' is not configured correctly.`);
}
`